### ISSUE TITLE
Build suddenly failing with "Review cannot be requested from pull request author."

### ISSUE THREAD

**Issue by Hannah-GBS** [peter-evans/create-pull-request#1835]
### Subject of the issue

As of this week my workflows (on 2 repos) have started failing with this error

![](https://i.imgur.com/ofinwEs.png)

The same workflow has worked fine for months prior to this without any changes, and removing the `reviewers: Hannah-GBS` line from my workflow appears to fix it, so mostly I'm just curious why this behaviour changed by itself.

**Comment by peter-evans** [comment]
Hi @Hannah-GBS 

I changed the behaviour of this in the last release, but it seems I failed to make mention of it in the release notes. Apologies for that.

In previous releases, if the pull request author was requested for a review it would fail because it isn't possible, but instead of an error I was just posting a warning in the log. I'm sure many users didn't realise the warning was there.

Anyway, I decided that this really should be an error, not a warning. So you've done the right thing and changed the `reviewers` input. 👍
